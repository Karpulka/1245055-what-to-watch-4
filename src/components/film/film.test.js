import React from "react";
import renderer from "react-test-renderer";
import Film from "./film.jsx";

const film = {
  title: `Film name`,
  src: `111`
};

it(`Render Film`, () => {
  const tree = renderer
    .create(<Film film={film} onFilmTitleClick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
