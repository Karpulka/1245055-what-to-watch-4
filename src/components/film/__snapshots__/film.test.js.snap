// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Film 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Film name"
      height="175"
      src="111"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Film name
    </a>
  </h3>
</article>
`;
